curl.md

```diff
@@ -6,4 +6,4 @@ A funny detail about Open Source projects is that they are called "projects",
 as if they were somehow limited in time or ever can get done. The cURL
 "project" is a number of loosely coupled individual volunteers working on
 writing software together with a common mission: to do reliable data transfers
-with Internet protocols. And giving away the code free for anyone to use.
+with Internet protocols, and give the code to anyone to use.
```

Comment:
For me, the "for free" part is a key that I don't think should be removed. The rest of the change I think is good!

Reply:
Great! Now if I can figure out how to go back and change that...still relatively new to Github.